src/agent/onefuzz-telemetry/src/lib.rs

```diff
@@ -63,7 +63,7 @@ impl fmt::Display for InstanceTelemetryKey {
     }
 }
 
-pub type TelemetryClient = appinsights::TelemetryClient<appinsights::InMemoryChannel>;
+pub type TelemetryClient = appinsights::blocking::TelemetryClient;
```

Comment:
Note, the InMemoryChannel is now default and not required to be specified as a type anymore.